pyi.Dockerfile

```diff
@@ -1,26 +1,44 @@
-# TODO: copied from the main Dockerfile... too bad we don't have includes.
-FROM ubuntu:20.04 as bcc-builder
+# Centos 7 image is used to grab an old version of `glibc` during `pyinstaller` bundling.
+# This will allow the executable to run on older versions of the kernel, eventually leading to the executable running on a wider range of machines.
+FROM centos:7 AS build-stage
```

Comment:
Any reason to not use holy-build-box?

Reply:
No reason. If I were to rewrite it, then maybe. HBB has caused me some headaches in other projects so I preferred to use mainline CentOS.